tox.ini

```diff
@@ -154,3 +154,34 @@ changedir=packages/bootstrap/extra
 commands=
   pip install .
   pytest --basetemp={envtmpdir} {posargs}
+
+
+[testenv:py35-collect-tests]
+platform=linux|darwin
+# See the following link for AWS_* environment variables:
+# http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials
+passenv =
```

Comment:
Why do these have to be set? Locally I can collect without these.

Reply:
I'm not sure if they have to, but I copied them from the other tox environments. I wasn't sure if them being set would make a difference.